core-bundle/src/Image/Studio/Figure.php

```diff
@@ -130,10 +130,17 @@ public function getSchemaOrgData(): array
             $imageIdentifier = '#/schema/image/'.$this->getMetadata()->getUuid();
         }
 
+        $imageSrc = $this->getImage()->getImageSrc();
+
+        // Workaround for Contao 4.13 only (see #6388)
+        if ('' !== $imageSrc && '/' !== $imageSrc[0] && !str_starts_with($imageSrc, 'http')) {
```

Comment:
```suggestion
        if ('' !== $imageSrc && '/' !== $imageSrc[0] && !preg_match('/^https?:/i', $imageSrc)) {
```

Otherwise URLs like `httpimages/foo.jpg` would not not get fixed.